Rakefile

```diff
@@ -0,0 +1,11 @@
+require "bundler/gem_tasks"
+
+require 'rspec/core/rake_task'
+
+task :default => :test
```

Comment:
If you are going to have more than one rake task you should probably not point default at test.  
If you want to have a default task this is a nice option.
```
task :default do
  system "rake --tasks"
end
```
This should be the equivalent of 
```
rake -T 
```